CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# CONTRIBUTING GUIDELINES
+::tada Welcome to Eddie Jaoude community, thank you for your interest in contributing to our projects. :balloon: To get started Please read the [Code of Conduct](https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/blob/develop/CODE_OF_CONDUCT.md)
+
+# EDDIE JAOUDE COMMUNITY PROJECTS
+Our projects are open to all contributors, to get started with contributing to our projects, you can check out our projects [here](https://github.com/EddieJaoudeCommunity), and choose a project you wish to make contributions to.
+
+# HOW DO I CONTRIBUTE TO A PROJECT?
+- If this is your first contribution, see [NEWCOMERS.md](). You might also want to take up issues with the [good first issue](https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) or [help wanted](https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/labels/help%20wanted) label.
+
+- Discuss the changes you wish to make by creating an [issue](https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/issues/new) or comment on an already created issue, or contact us via our [discord channel](https://discord.com/invite/jZQs6Wu). 
+- Once, you have been assigned the issue by the maintainer, you can go ahead to fork the repo, clone and make changes and fix the issue. For steps on how to do this, this [here]() is a helpful guide.
```

Comment:
For the steps link, I have linkedin to GitHub docs for now, what do you think?

```suggestion
- Once, you have been assigned the issue by the maintainer, you can go ahead to fork the repo, clone and make changes and fix the issue. For steps on how to do this, this [here](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) is a helpful guide.
```

Reply:
I think a link to the newcomers.md would be preferable.

Reply:
Sure 👍